conjur-api/Policy.cs

```diff
@@ -0,0 +1,45 @@
+﻿// <copyright file="Policy.cs" company="Conjur Inc.">
+//     Copyright (c) 2016 Conjur Inc. All rights reserved.
+// </copyright>
+// <summary>
+// Conjur Policy entity
+// </summary>
+namespace Conjur
+{
+    using System.IO;
+    using System.Net;
+
+    public class Policy : Resource
+    {
+        private readonly string path;
+
+        internal Policy(Client client, string name)
+                : base(client, Constants.KIND_POLICY, name)
+        {
+            this.path = $"policies/{WebUtility.UrlEncode(client.GetAccountName())}/{Constants.KIND_POLICY}/{WebUtility.UrlEncode(name)}";
```

Comment:
This is mostly preference, so not a requested change but something to consider generally. With long path concatenations like this, it can be helpful to make each part a little easier to visually parse out when reading it. One example I though of is:
```csharp
this.path = String.Join("/", new string[] {
  "policies",
  WebUtility.UrlEncode(client.GetAccountName()),
  Constants.KIND_POLICY,
  WebUtility.UrlEncode(name)
});
```
You could also move this string formatting and encoding into a helper method so that you can just have something like:
```csharp
this.path = BuildPath("policies", client.GetAccountName(), Constants.KIND_POLICY, name);
```
Basically anything that helps make this line less about string formatting and more about what is going into the URL makes it easier to review and read later.

Reply:
Thanks - I would like to update those string to URI and use a builder someday